templates/Dockerfile.j2

```diff
@@ -0,0 +1,56 @@
+# This Dockerfile was generated from templates/Dockerfile.j2
+{% set tarball = 'logstash-%s.tar.gz' % elastic_version -%}
+{% if '-' in version_tag -%}
+{%   set url_root = 'https://staging.elastic.co/%s/downloads/logstash' % version_tag -%}
+{%   set pack_url = 'https://staging.elastic.co/%s/downloads/logstash-plugins' % version_tag -%}
+{% else -%}
+{%   set url_root = 'https://artifacts.elastic.co/downloads/logstash' -%}
+{%   set pack_url = 'https://artifacts.elastic.co/downloads/logstash-plugins' -%}
+{% endif -%}
+
+
+FROM centos:7
+MAINTAINER Elastic Docker Team <docker@elastic.co>
+
+# Install Java and the "which" command, which is needed by Logstash's shell
+# scripts.
+RUN yum install -y java-1.8.0-openjdk-headless which && \
+    yum clean all
+
+# Provide a non-root user to run the process.
+RUN groupadd --gid 1000 logstash && \
+    adduser --uid 1000 --gid 1000 \
+      --home-dir /usr/share/logstash --no-create-home \
+      logstash
+
+# Add Logstash itself.
+RUN curl -Lso - {{ url_root }}/{{ tarball }} | \
```

Comment:
Is it possible to remove the `s` parameter to help diagnose slowness issues with interactive builds? While builds are run in an automated fashion, still, unified release processes will be triggered manually and it's beneficial to see if something is stuck due to slow network.